Editor/EditorCore/ProBuilderAnalytics.cs

```diff
@@ -11,29 +11,55 @@
 //#define PB_ANALYTICS_DONTSEND
```

Comment:
Can we add `#if !DEBUG` here to exclude developer builds?

Reply:
I added that in [40fa1de](https://github.com/Unity-Technologies/com.unity.probuilder/pull/475/commits/40fa1de3f1d9570b95e15929a997332213ac3a60) but at the Send event level so that we can still log analytics in a file to test but they are not sent to the analytics for real :D
Does that sound ok ?